explainer.md

```diff
@@ -913,6 +913,29 @@ xrSession.depthNear = 1.0;
 xrSession.depthFar = 100.0;
 ```
 
+### Handling non-opaque displays
+
+Some devices which support the WebXR Device API may use displays that are not fully opaque, or otherwise show the real world in some capacity. To determine how the display will blend rendered content with the real world, check the `XRSession`'s `worldBlendMode` attribute. It may currently be one of three values, and more may be added in the future if new display technology necessitates it:
+
+  - `opaque`: The world is not visible at all through this display. Transparent pixels in the `baseLayer` will appear black. This is the expected mode for most VR headsets.
+  - `additive`: The world is visible through the display and pixels in the `baseLayer` will be shown additively against it. Black pixels will appear fully transparent. Alpha values will modulate the intensity of the colors but cannot make a pixel opaque. This is the expected mode for devices like HoloLens or Magic Leap.
```

Comment:
For native apps on HoloLens, alpha values are ignored in the headset display.  Can you elaborate on how a UA might use alpha values to modulate the intensity of colors?

Reply:
I actually thought this WAS how HoloLens worked, with the colors coming through as sort of pre-multiplied by the alpha channel. I'll correct the PR.